src/service-workers/firebase-messaging/notifications.ts

```diff
@@ -0,0 +1,240 @@
+      const didFail = failed === 'true'
+      return {
+        title: `Transaction ${didFail ? 'failed' : 'executed'}`,
+        body: `Safe ${shortenAddress(address)} on ${chainName} ${
+          didFail ? 'failed to execute' : 'executed'
+        } transaction ${shortenAddress(txHash)}.`,
+      }
+    },
+    [WebhookType.PENDING_MULTISIG_TRANSACTION]: ({ address, safeTxHash }) => {
+      return {
+        title: 'Pending transaction',
+        body: `Safe ${shortenAddress(address)} on ${chainName} has a pending transaction ${shortenAddress(
+          safeTxHash,
+        )}.`,
+      }
+    },
+    [WebhookType.INCOMING_ETHER]: ({ address, txHash, value }) => {
+      return {
+        title: `${currencyName} received`,
+        body: `Safe ${shortenAddress(address)} on ${chainName} received ${formatUnits(
+          value,
+          chain?.nativeCurrency?.decimals,
+        ).toString()} ${currencySymbol} in transaction ${shortenAddress(txHash)}.`,
+      }
+    },
+    [WebhookType.OUTGOING_ETHER]: ({ address, txHash, value }) => {
+      return {
+        title: `${currencyName} sent`,
+        body: `Safe ${shortenAddress(address)} on ${chainName} sent ${formatUnits(
+          value,
+          chain?.nativeCurrency?.decimals,
+        ).toString()} ${currencySymbol} in transaction ${shortenAddress(txHash)}.`,
+      }
+    },
+    [WebhookType.INCOMING_TOKEN]: async ({ address, txHash, tokenAddress, value }) => {
+      const token = await getTokenInfo(data.chainId, address, tokenAddress, value)
+      return {
+        title: `${token.name} received`,
+        body: `Safe ${shortenAddress(address)} on ${chainName} received ${token.value} ${
+          token.symbol
+        } in transaction ${shortenAddress(txHash)}.`,
+      }
+    },
+    [WebhookType.OUTGOING_TOKEN]: async ({ address, txHash, tokenAddress, value }) => {
+      const token = await getTokenInfo(data.chainId, address, tokenAddress, value)
+      return {
+        title: `${token.name} sent`,
+        body: `Safe ${shortenAddress(address)} on ${chainName} sent ${token.value} ${
+          token.symbol
+        } in transaction ${shortenAddress(txHash)}.`,
+      }
+    },
+    [WebhookType.MODULE_TRANSACTION]: ({ address, module, txHash }) => {
+      return {
+        title: 'Module transaction',
+        body: `Safe ${shortenAddress(address)} on ${chainName} executed a module transaction ${shortenAddress(
+          txHash,
+        )} from module ${shortenAddress(module)}.`,
+      }
+    },
+    [WebhookType.CONFIRMATION_REQUEST]: ({ address, safeTxHash }) => {
+      return {
+        title: 'Confirmation request',
+        body: `Safe ${shortenAddress(
+          address,
+        )} on ${chainName} has a new confirmation request for transaction ${shortenAddress(safeTxHash)}.`,
+      }
+    },
+    [WebhookType.SAFE_CREATED]: () => {
+      // We do not preemptively subscribe to Safes before they are created
+      return null
+    },
+  }
```

Comment:
This could be moved to a separate module as a static config.

Reply:
Moved in a115c14973ce9cf0f7b6af14d7ae633ef6af035d.